.gitmodules

```diff
@@ -15,3 +15,6 @@
 [submodule "lib/sdl/SDL_ttf"]
 	path = lib/sdl/SDL_ttf
 	url = https://github.com/SDL-mirror/SDL_ttf
+[submodule "lib/libcxx"]
+	path = lib/libcxx
+	url = https://github.com/thrimbor/nxdk-libcxx.git
```

Comment:
This should move to XboxDev first (ideally without the "threads" branch). The nxdk branch should be made the default then.

Reply:
Agree. Any update here?

Reply:
Done.